src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/TicketCount.java

```diff
@@ -32,5 +36,9 @@ public class TicketCount {
 
 	@Column
 	private boolean fineReductionRequest;
+	
+	@JsonBackReference
+	@ManyToOne(targetEntity=Dispute.class, fetch = FetchType.LAZY)
+	private Dispute dispute;
```

Comment:
Is there a way to hide this property from the generated swagger def?  If I exclude the "id"  and "dispute" properties from a Postman POST call to /api/v1.0/dispute, I'm able to create a Dispute just fine.
```
  "ticketCounts": [
    {
      "offenceDeclaration": "string",
      "timeToPayRequest": true,
      "fineReductionRequest": true
    },
    {
      "offenceDeclaration": "string",
      "timeToPayRequest": true,
      "fineReductionRequest": true
    }
  ],
```